[2023-10-03T12:30:19.862+0000] {taskinstance.py:1119} INFO - Dependencies not met for <TaskInstance: launches_getter.check_api_status scheduled__2023-09-27T00:00:00+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2023-10-03T12:30:19.916+0000] {local_task_job_runner.py:161} INFO - Task is not able to be run
